extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -69,28 +71,48 @@ public function header() {
 	 * @return array
 	 */
 	public function getTopContributorsData() {
-		$userContribCount = $this->usersModel->getUserContributions( $this->getUser() );
+		$userContributionCount = $this->usersModel->getUserContributions( $this->getUser() );
 		$contributors = CommunityPageSpecialUsersModel::filterGlobalBots(
 				// get extra contributors so if there's global bots they can be filtered out
-				CommunityPageSpecialUsersModel::getTopContributors( 50, '1 MONTH' )
+				CommunityPageSpecialUsersModel::getTopContributors( 50 )
 			);
 		// get details for only 5 of the remaining contributors
 		$contributorDetails = $this->getContributorsDetails( array_slice( $contributors, 0, 5 ) );
 
+		$userRank = '-';
+		$editors = count( $contributors );
+
+		if ($editors == 0) {
```

Comment:
Minor: Always prefer strict comparisons (`$editors === 0`) unless you really need it to be not strict, and also should be spacey per conventions.